client/packages/system/src/Patient/Encounter/CreateEncounterModal.tsx

```diff
@@ -87,12 +87,7 @@ export const CreateEncounterModal: FC = () => {
       ...draft,
       startDatetime,
       createdBy: { id: user?.id ?? '', username: user?.name ?? '' },
-      status:
-        date && DateUtils.isFuture(date)
-          ? EncounterNodeStatus.Scheduled
-          : date && DateUtils.isPast(date)
-          ? EncounterNodeStatus.Missed
-          : draft?.status,
+      status: EncounterNodeStatus.Pending,
```

Comment:
So there's no different default if setting a new encounter in the past now?

Reply:
No, the statuses Missed and Scheduled must  be calculated on the fly now and an encounter can be marked accordingly (in a future PR). The reason is that you would have to somehow update the encounter statues automatically, e.g. in a background process. Moreover, one idea of the patient program is that the clinician is in control of all document changes but an automated status setter would need to change the encounter document without user interaction.